sway/tree/view.c

```diff
@@ -141,6 +141,9 @@ static void view_autoconfigure_floating(struct sway_view *view) {
 	view->border_top = view->border_bottom = true;
 	view->border_left = view->border_right = true;
 
+	// Don't maximize floating windows
+	view_set_tiled(view, false);
```

Comment:
Probably want to enable tiling in `view_autoconfigure`, in case it's not floating